lib/rag-engines/aurora-pgvector/index.ts

```diff
@@ -30,7 +30,13 @@ export class AuroraPgVector extends Construct {
       engine: rds.DatabaseClusterEngine.auroraPostgres({
         version: rds.AuroraPostgresEngineVersion.VER_15_3,
       }),
-      removalPolicy: cdk.RemovalPolicy.DESTROY,
+      storageEncryptionKey: props.shared.kmsKey,
+      // Always setting it to true would be a breaking change.
+      storageEncrypted: props.shared.kmsKey ? true : false,
```

Comment:
My preference would be to always enable it but it would trigger the recreation of the table